Formula/virtualpg.rb

```diff
@@ -1,8 +1,8 @@
 class Virtualpg < Formula
   desc "Loadable dynamic extension for SQLite and SpatiaLite"
   homepage "https://www.gaia-gis.it/fossil/virtualpg/index"
-  url "https://www.gaia-gis.it/gaia-sins/virtualpg-1.0.2.tar.gz"
-  sha256 "37abd56b2899c90b355e50d843ba019af540b83e7af1da110a766f041abb514e"
+  url "https://www.gaia-gis.it/gaia-sins/virtualpg-2.0.1.tar.gz"
+  sha256 "be2aebeb8c9ff274382085f51d422e823858bca4f6bc2fa909816464c6a1e08b"
 
```

Comment:
```suggestion
  license `MPL-1.1`
```

Reply:
let's do this in a separate PR along with livecheck as well :)

Reply:
#69101

Reply:
Thanks @Moisan!